src/dawn/IIR/Stage.cpp

```diff
@@ -32,7 +32,7 @@ Stage::Stage(StencilInstantiation& context, int StageID)
 
 Stage::Stage(StencilInstantiation& context, int StageID, const Interval& interval)
     : stencilInstantiation_(context), StageID_(StageID) {
-  insertChild(make_unique<DoMethod>(interval));
+  insertChild(make_unique<DoMethod>(interval, context));
```

Comment:
is context the good name since we have an OptimizerContext?

Reply:
yes, ok, I can change it here